client/src/docs-ui/repo-actions/repo-actions.tsx

```diff
@@ -11,10 +11,10 @@ export class DocsRepoActions {
     return (
       <Host class={repoActionsStyle}>
         <amplify-external-link
-          href={`https://github.com/aws-amplify/docs/issues/new?title=[feedback]&labels=v2&body=${encodeURI(
-            `**Page**: [\`${location.href}\`](${location.href})
+          href={`https://github.com/aws-amplify/docs/issues/new?title=Feedback&body=${encodeURI(
```

Comment:
I think the square bracket here is better, it should be a capital 'F' tho. Also you should make this common if possible since it is used in multiple places.

Reply:
Agree square brackets are nice

Reply:
Additionally, I believe this needs to change in `feedback-callout`. All the more reason to have a common const or function for this